docs/index.md

```diff
@@ -54,16 +54,25 @@ We have specific instructions for certain projects and environments. If you are
 
 ### Configuring CodeSee with Create React App
 
-**Add React App Rewired**
+Add [React App Rewired](https://github.com/timarney/react-app-rewired#how-to-rewire-your-create-react-app-project) to your project as described below:
 
```

Comment:
I think it's confusing to both link to their instructions, and copy them below. Won't some people click the link and follow that?

Can we be more explicit about that (note I changed the link so it no longer points at the install instructions):

> We're going to add [React App Rewired](https://github.com/timarney/react-app-rewired) following the instructions below. Please note that these instructions are slightly different than the ones on the react-app-rewired site. But, if you already have react-app-rewired installed, nothing to worry about here, you can skip to the next step.